dts/bindings/i2c/ti,mspm0-i2c.yaml

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI MSPM0 I2C
+
+compatible: "ti,mspm0-i2c"
+
+include: [i2c-controller.yaml, pinctrl-device.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  pinctrl-0:
+    required: true
+
+  pinctrl-names:
+    required: true
+
+  primary-address:
+    type: int
+    required: false
+
+  secondary-address:
```

Comment:
NACK! Am aware that this IP supports 2 target at the same time and there is no such support available in zephyr so far. But this shall only be part of target register driver. For example, 1 eeprom and 1 sensor, both registering independently. I2C driver shall handle this based on available counts or array type statically defined or dynamically checked with register set.